atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/TargetedSweepEteTest.java

```diff
@@ -93,6 +93,15 @@ public void targetedSweepLargeStreamsTest() {
         assertDeletedAndSwept(4, 4, 4, 4 * 4);
     }
 
+    @Test
+    public void targetedSweepCleanupUnmarkedStreamsTest() {
+        todoClient.storeUnmarkedSnapshot("snap");
+        todoClient.storeUnmarkedSnapshot("crackle");
+        todoClient.storeUnmarkedSnapshot("pop");
+        todoClient.runIterationOfTargetedSweep();
+        assertDeleted(0, 3, 3, 3);
```

Comment:
lol wat????

Reply:
```
    private void assertDeleted(long idx, long hash, long meta, long val) {
        Assert.assertThat(todoClient.numberOfCellsDeleted(INDEX_TABLE), equalTo(idx));
        Assert.assertThat(todoClient.numberOfCellsDeleted(HASH_TABLE), equalTo(hash));
        Assert.assertThat(todoClient.numberOfCellsDeleted(METADATA_TABLE), equalTo(meta));
        Assert.assertThat(todoClient.numberOfCellsDeleted(VALUES_TABLE), equalTo(val));
    }
```

It's kind of horrible but I didn't really fancy inlining everything in to the existing tests. I'll leave a comment why the numbers are as they are